src/main/java/com/revature/pojos/BasicUser.java

```diff
@@ -2,9 +2,21 @@
 
 import java.util.List;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="BASIC_USER")
 public class BasicUser extends User{
+	
+	@Column(name="FIRST_NAME")
 	private String firstName;
+	
+	@Column(name="LAST_NAME")
 	private String lastName;
+	
+	
```

Comment:
can you put a // TODO here so we know we gotta go back to it

Reply:
sure